### DISCUSSION TITLE
Need to a sample input LAMMPS file for NPT MD

### DISCUSSION THREAD

**Discussion by saeedshahi** [deepmodeling/deepmd-kit#1259] [Q&A] [answered]
Hello all,

There is a sample lammps input file in the package but it is for NVT MD, I need  NPT ones. So, if you can provide me ones I appreciated. I could not use a SHAKE constrained with LAMMPS deepmd. Is it right? 

Best,
Saeed

**Comment by wanghan-iapcm** [comment, marked as answer]
You can consult lammps manual on `fix npt` https://docs.lammps.org/fix_nh.html#fix-npt-command

You do not need to use SHAKE because deepmd-kit does not assume any constrained bond.
